packages/visualizations/src/components/utils/MiniMap.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import tippy from './tippy';
+    import SvgChoropleth from '../Map/Svg/Choropleth.svelte';
+    import type { ColorScale } from '../types';
+    import type { ChoroplethDataValue, NavigationMap } from '../Map/types';
+
+    export let active: boolean;
+    export let showTooltip: boolean;
+    export let data: { value: ChoroplethDataValue[] };
+    export let map: NavigationMap;
+    export let colorScale: ColorScale | undefined;
+
+    $: ({ label } = map);
+    $: options = { shapes: map.shapes, colorScale };
+</script>
+
+{#if showTooltip}
+    <button
+        type="button"
+        class:active
+        on:click
```

Comment:
`on:click` is unused ? To delete ?

Reply:
I guess it's useful as it allows to pass a callback to the component. Without it doesn't seem to work.

Reply:
I was too quick. 😅 
This is what you can find in the Svelte doc: 

> https://svelte.dev/docs#template-syntax-element-directives-on-eventname 
>
> If the on: directive is used without a value, the component will forward the event, meaning that a consumer of the component can listen for it.